src/main/resources/explorer/assets/plugin-explorer.css

```diff
@@ -40,18 +40,20 @@ li.active .file-tree {
 }
 .folder-expander octicon {
   font-size: 0.8em;
+  color: #fff
 }
 .root-expander {
   position: absolute;
-  padding: 2px 2px 0 5px;
-  right: 5px;
+  padding: 2px 0px 0 4px;
+  right: 15px;

```

Comment:
The icon content alignment when expanded is little bit off by these style.
And I think the button and badges should be aligned by center rather than right.
The below styles looks good for me.
```
  padding: 0;
  width: 25px;
  height: 25px;
  right: 10px;
```